x-pack/plugin/fleet/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/fleet/40_secrets_get.yml

```diff
@@ -3,6 +3,10 @@
       fleet.post_secret:
         body: '{"value": "test secret"}'
   - set:  { id: id }
+  # search node needs to be available for fleet.get_secret to work
```

Comment:
nit Maybe add "in stateless" somewhere in the comment.

Reply:
will do.